ocptv/output/config.py

```diff
@@ -93,6 +117,9 @@ def config(
     if enable_runtime_checks is not None:
         _config.enable_runtime_checks = enable_runtime_checks
 
+    if timezone is not _NOT_SET:
```

Comment:
If `ty.TYPE_CHECKING` is `True` then `_NOT_SET` would be `None` and one would not be able to reset the timezone by using `None` as the docstring states, right?

Reply:
`ty.TYPE_CHECKING` is true only when... type checking. So everything in that branch of the if doesnt affect runtime behavior. I had to introduce that `_NOT_SET` because `None` is a valid value for the parameter, which makes `ty.Optional` unusable